cloudinit/sources/DataSourceOVF.py

```diff
@@ -99,9 +101,7 @@ def _get_data(self):
             if not self.vmware_customization_supported:
                 LOG.debug("Skipping the check for "
                           "VMware Customization support")
-            elif not util.get_cfg_option_bool(
-                    self.sys_cfg, "disable_vmware_customization", True):
```

Comment:
It appears that you have dropped the ability to disable vmware customization. This should be considered very carefully, as the previous value was `True`.  I don't have resources (time or vmware) to fully consider this, but it feels like this is a big change.

You'll need to comment on why that is OK, and adjust tests that reference it (tests/unittests/test_datasource/test_ovf).  If you're dropping the configuration setting entirely, then you'd want to remove all references to it across the code base.